mmv1/api/resource.go

```diff
@@ -552,9 +556,6 @@ func (r Resource) UserVirtualFields() []*Type {
 }
 
 func (r Resource) ServiceVersion() string {
-	if r.CaiBaseUrl != "" {
```

Comment:
I don't find the cases that CaiBaseUrl starting with a specific version and it shouldn't.